### ISSUE TITLE
VS Code doesn't recognizes Fira Code (duplicating #889)

### ISSUE THREAD

**Issue by cangyin** [tonsky/FiraCode#1116]
I installed FiraCode-VF.ttf version 5.2.
Font settings in my VSCode are as follows:
```json
    "editor.fontFamily": "Fira Code",
    "editor.fontLigatures": true
```
I also tried 
```json
    "editor.fontFamily": "'Fira Code'",
    "editor.fontLigatures": true
```
or
```json
    "editor.fontFamily": "Fira Code Bold",
    "editor.fontLigatures": true
```
and restarted VSCode and my system again and again. It just dosn't work.

![image](https://user-images.githubusercontent.com/50256182/92227396-d01b0600-eed8-11ea-8a7c-ef2560072823.png)

**Comment by NerdyBread** [comment]
Yeah same thing happened to me

**Comment by tonsky** [comment]
Try using non-VF version? Most programs do not support it properly. I am not sure if that’s of VS Code / Windows, but it might be

**Comment by wopian** [comment]
> Try using non-VF version? Most programs do not support it properly. I am not sure if that’s of VS Code / Windows, but it might be

Just tried it with only VF installed using 5.2 on VS Code 1.48 on Windows 10

- `Fira Code` - Shows Fira Code (FC)
- `Fira Code Thin` - Shows FC Thin
- `Fira Code Regular` - Shows FC Regular
- `Fira Code Medium` - Shows FC Medium
- `Fira Code SemiBold` - Shows FC SemiBold
- `Fira Code Bold` - **Does not display Fira Code**
- `Fira Code Retina` - **Does not display Fira Code**

**Comment by cangyin** [comment]
@wopian My VSCode font settings does't appear like that of yours. I have to guess the font name and fill it by hand, which also confuses me.
![image](https://user-images.githubusercontent.com/50256182/92344851-21eea680-f0fa-11ea-9c19-01bdba92c124.png)

**Comment by Boolean-ball** [comment]
@cangyin   
 Your VScode Font family setting is wrong! Is "**'Fira code'**,Consolas...",have a space...so need single quotes.

**Comment by cangyin** [comment]
@Boolean-ball  I tried with single quote, and it doesn't work.

**Comment by wopian** [comment]
@cangyin that's the Windows 10 font settings screen for Fira Code.

**Comment by Hocnonsense** [comment]
I don't use quote but it do work.
Just copy & paste this?
```
Fira Code, Microsoft YaHei mono, monospace
```

**Comment by KUndn** [comment]
@Hocnonsense 



> I don't use quote but it do work.
> Just copy & paste this?
> 
> ```
> Fira Code, Microsoft YaHei mono, monospace
> ```
@Hocnonsense thanks man its working now

**Comment by protoEvangelion** [comment]
I just had to reload vscode.

**Comment by mattdeann** [comment]
DId you open the ttf folder and install the fonts?

**Comment by cangyin** [comment]
@mattdeann Yeah! It works now. I installed all .ttf files in `ttf` folder (as before I just installed the .ttf in `variable-ttf` folder). But I don't know why this is the solution.

**Comment by Pinche-Dev** [comment]
"editor.fontLigatures": true,
worked for me

**Comment by AmamiyaRenn** [comment]
I rebooted my PC, and it works(
